backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepository.java

```diff
@@ -52,6 +52,7 @@
 	private Map<UUID, List<Token>> shopIdToListOfTokensMap = new HashMap<>();
 	private Map<UUID, AtomicInteger> shopIdToLastAllotedTokenMap = new HashMap<>();
 	private Map<String, ShopOwner> shopOwnerMap = new HashMap<>();
+	private final static long WAITING_TIME_MINS = 4L;
```

Comment:
Nit. 'static final' instead of 'final static' ?

Reply:
Done.